packages/legend-query-builder/src/components/QueryBuilderResultPanel.tsx

```diff
@@ -272,8 +272,30 @@ const QueryBuilderGridResultContextMenu = observer(
           );
     };
 
+    const handleCopyCellValue = applicationStore.guardUnhandledError(() =>
+      applicationStore.copyTextToClipboard(event?.value),
```

Comment:
huh, I don't get it, where is this `event` coming from?

Reply:
Thanks @YannanGao-gs! Looking great here. @MauricioUyaguari what do you think about the 2nd video? Should we stringify the data and produce CSV in the copy content or should we leave it as JSON like shown?

Reply:
If we have copy row value, won't people want copy column values as well?
As for the format, probably csv is the best form to copy for users to understand.

Reply:
Thinking about it further, copy column values is a bit too risky. Im cautious about adding copy functions due to data leakage. We have warnings for export to csv.
So for now just copy cell and copy row value are okay.